### PR TITLE
Handel to device issue and Fix empty user_name column in db

### PR DESCRIPTION
**Pull request by JerryYu0815**

### CODE REVIEW

File: techeria_controller/templates/index.html [hunk 1]
```diff
@@ -109,10 +109,19 @@
                                 </div>
                             </div>
 
+                            {% if user.is_authenticated %}
```
Comment (lianahasan) [inline techeria_controller/templates/index.html:112]: should it also check is_staff

Comment (farahh001) [inline techeria_controller/templates/index.html:112]: We need to make sure the user authentication works in all pages

File: techeria_controller/techeria_app/views.py [hunk 2]
```diff
@@ -218,10 +218,9 @@ def registration(request):
 
 
                 if category == "Buyer":
-                    buyer.username = username
+                    buyer.user_name = username
```
Comment (farahh001) [inline techeria_controller/techeria_app/views.py:221]: it's good that you have noticed the mistake and fix the user_name

File: techeria_controller/techeria_app/views.py [hunk 3]
```diff
@@ -263,7 +262,7 @@ def registration(request):
                     seller.user_name = username
                     seller.first_name=first_name
                     seller.last_name=last_name
-                    seller.username=username
+                    seller.user_name=username
```
Comment (kabirulislam95) [inline techeria_controller/techeria_app/views.py:265]: Thanks for making cchanges

**Review by sanjidanisha — COMMENTED** [review]
looks good

**Pull request merged**